src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Providers/PsBackupProviderManager.cs

```diff
@@ -138,6 +138,9 @@ public IPsBackupProvider GetProviderInstance(WorkloadType workloadType)
                 case WorkloadType.AzureVM:
                     providerType = PsBackupProviderTypes.IaasVm;
                     break;
+                case WorkloadType.AzureFiles:
```

Comment:
WorkloadType.AzureFiles and PsBackupProviderTypes.AzureFile -> can you please normalize this?